### ISSUE TITLE
Make a version of the forum that is available to anonymous users.

### ISSUE THREAD

**Issue by klondikemarlen** [elthran/UndyingKingdoms#309]
This would allow users to make a bug report if they can't log in.

**Comment by elthran** [comment]
I would rather we don't do this (because EVEN if we want forum to be viewable while not logged in, I don't want posting able without a proper user). I'd rather have a contact us / support page to report an issue like not being able to log in.

**Comment by klondikemarlen** [comment]
Yeah ... like stack overflow. You can't post with an account but you can still read stuff. Probably won't happen soon though? I'll keep it in mind though as I'm redoing currently redoing the forum in https://github.com/elthran/UndyingKingdoms/tree/issue-225.
